natss/config/README.md

```diff
@@ -22,7 +22,7 @@ They do not offer:
 ## Deployment steps
 
 1. Setup
-   [Knative Eventing](https://knative.dev/eventing/blob/master/DEVELOPMENT.md).
+   [Knative Eventing](https://knative.dev/docs/eventing/getting-started/).
```

Comment:
Correct link is https://github.com/knative/eventing-contrib/blob/master/DEVELOPMENT.md ?

Change to 
```
   [Knative Eventing](https://github.com/knative/eventing-contrib/blob/master/DEVELOPMENT.md).
```

Reply:
I quite honestly kind of forgot the PR. I'll just close the pull request for now.

I'll fight the CLA thing at a later time if I have something a bit more valuable in a PR.
For now I assume (hope) the link will be fixed anyhow.